javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,184 @@
+    switch (result) { // switch statement 
+        case 0:
+            console.log(daysWeek[1]);
+            break;
+        case 1:
+            console.log(daysWeek[2]);
+            break;
+        case 2:
+            console.log(daysWeek[3]);
+            break;
+        case 3:
+            console.log(daysWeek[4]);
+            break;
+        case 4:
+            console.log(daysWeek[5]);
+            break;
+        case 5:
+            console.log(daysWeek[6]);
+            break;
+        case 6:
+            console.log(daysWeek[0]);
+            break;
+        }
+        if (isNaN(result) || days === "") {// test for missing argument
+            console.log("Please, enter a date")
+        }
+        return result;
+    
+    }
+    
+    getEventWeekday(0);
+    
+    //Weather wear
+    function getDressed(temp) {
+    
+        if (temp >= 35 || temp <= -20) {
+            console.log("It is better to stay home.");
+        } else if (temp >= 20 && temp < 35) {
+            console.log("shorts and a t-shirt");
+        } else if (temp >= 10 && temp < 20) {
+            console.log("some jeans and a jaket");
+        } else if (temp > -20 && temp < 10) {
+            console.log("It is a sweater weather");
+        } else if (temp === undefined || temp === "") {
+            console.log("Please, enter a temperature");
+        }
+        return temp;
+    }
+    const clothesToWear = getDressed();
+
+//Student manager
+const class07Students = [];
+
+
+function addStudentToClass(studentName) {
+
+    if (!class07Students.includes(studentName) // excludes duplicates
+    && studentName != ``) { //  and does not add an empty string
+        class07Students.push(studentName);
+    } else if (class07Students.includes(studentName)) { // if includes duplicats does not add it
+        console.log(`Student ${studentName} is already in the class`);
+    }
+
+    if (class07Students.length <= 6 
+        && studentName === "Queen") { // always space for Queen
+            class07Students.push();
+        } else if (class07Students.length > 6 
+        && studentName !== "Queen") { // otherwise max.length = 6
+            class07Students.length = 6;
+            console.log("Cannot add more students to class 07");
+        } 
+
+    return studentName;
+}
+
+addStudentToClass(""); 
+addStudentToClass("Anna"); // 1
+addStudentToClass("Maria"); // 2
+addStudentToClass("Viktor"); // 3
+addStudentToClass("Peter"); // 4
+addStudentToClass("Viktoria"); // 5
+addStudentToClass("Nelia"); // 6
+addStudentToClass("Christian"); // Cannot add more students to class 07
+addStudentToClass("Mathilda"); // Cannot add more students to class 07
+addStudentToClass("Peter"); //  Student Peter is already in the class
+addStudentToClass("Queen"); // adds Queen
+
+
+console.log(class07Students)
+
+function getNumberOfStudents(number) {
+  number = class07Students.length;
+  return number;
+}
+console.log(getNumberOfStudents());
```

Comment:
Based on the fact that you are not even using the number and not providing it to a function, I would say that you can simplify it by just returning class07Students.length;